src/components/HomePage/SearchResultsArea/SearchResultsWidget/SearchURLWidget.js

```diff
@@ -0,0 +1,65 @@
+import React, { useCallback } from "react";
+
+import { useDispatch } from "react-redux";
+import { makeStyles, Button, Typography, Divider } from "@material-ui/core";
+
+import { addSnackbar } from "../../../../actions/notificationActions";
+
+const useSearchURLWidgetStyles = makeStyles((theme) => ({
+    root: {
+        paddingTop: theme.spacing(3),
+    },
+    label: {
+        textAlign: "center",
+    },
+    container: {
+        margin: "10px auto",
+        paddingBottom: "10px",
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+        gap: "10px",
+    },
+    link: {
+        width: "95%",
+        fontWeight: "bold",
+        userSelect: "none",
+        whiteSpace: "nowrap",
+        overflowX: "scroll",
+        textAlign: "center",
+    },
+}));
+
+const SearchURLWidget = () => {
+
+    const classes = useSearchURLWidgetStyles();
+    const dispatch = useDispatch();
+
+    const fullURL = window.location.href;
+
+    const copyToClipboard = useCallback(() => {
+        navigator.clipboard.writeText(fullURL);
+
+        dispatch(addSnackbar({
+            message: "Copied to clipboard",
+            options: {
+                autoHideDuration: 1500,
+            },
```

Comment:
Did you manage to only change the duration in this snackbar?

Reply:
When I tested locally it worked as expected but then on another occasion it was showing some weird behavior, gotta look further into it.